public/app/features/dashboard-scene/scene/VariableControls.tsx

```diff
@@ -0,0 +1,70 @@
+import { css } from '@emotion/css';
+
+import { VariableHide } from '@grafana/data';
+import { selectors } from '@grafana/e2e-selectors';
+import { sceneGraph, useSceneObjectState, SceneVariable, SceneVariableState, ControlsLabel } from '@grafana/scenes';
+
+import { DashboardScene } from './DashboardScene';
+
+export function VariableControls({ dashboard }: { dashboard: DashboardScene }) {
+  const variables = sceneGraph.getVariables(dashboard)!.useState();
+
+  return (
+    <>
+      {variables.variables.map((variable) => (
+        <VariableValueSelectWrapper key={variable.state.key} variable={variable} />
+      ))}
+    </>
+  );
+}
+
+interface VariableSelectProps {
+  variable: SceneVariable;
+}
+
+export function VariableValueSelectWrapper({ variable }: VariableSelectProps) {
+  const state = useSceneObjectState<SceneVariableState>(variable, { shouldActivateOrKeepAlive: true });
```

Comment:
Nit: I think we need to add a params object to `sceneObject.useState()` to avoid doing this.